e2e/run.sh

```diff
@@ -65,6 +65,7 @@ kubectl run --rm \
   --env="VAULT_URL=${VAULT_URL:-}" \
   --env="GITLAB_TOKEN=${GITLAB_TOKEN:-}" \
   --env="GITLAB_PROJECT_ID=${GITLAB_PROJECT_ID:-}" \
+  --env="GITLAB_ENVIRONMENT=${GITLAB_ENVIRONMENT:-}" \
```

Comment:
I think all the `GITLAB_*` variables are unused currently and can be removed. In fact we don't run any e2e tests against the gitlab provider.

Reply:
removed